workflow-job

```diff
@@ -363,7 +405,8 @@ flows["${sha1}"] = {
             deleteContainer(containers.cassandra)
             deleteContainer(containers.elasticSearch)
             deleteContainer(containers.james)
-            deleteImage(images.james)
+            deleteImage(images.jamesCassandra)
+            deleteImage(images.jamesJPA)
             deleteContainer(containers.jamesJava6)
```

Comment:
Yes, we published to the docker hub and then we delete that docker image on docker machine. Is that correct?